Known_Issues/Service Fabric Common Name Digicert Multiple Issuer Thumbprints.md

```diff
@@ -0,0 +1,130 @@
+# Service Fabric Common Name Digicert Multiple Issuer Thumbprints
+
+[Issue](#Issue)  
+[Affects](#Affects)  
+[Symptoms](#Symptoms)  
+[Cause](#Cause)  
+[Impact](#Impact)  
+[Mitigation](#Mitigation)  
+[Resolution](#Resolution)  
+
+## Issue
+
+Service Fabric clusters using Certificate Common Name and Issuer thumbprint 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4 may become unresponsive or have state 'Upgrade Service Unreachable'.  
+
+## Affects
+
+This issue affects any cluster version that has the following configuration:  
+
+- Certificate Common Name instead of thumbprint
+- Certificate issued from Digicert
+- Certificate Issuer Thumbprint configured with only: 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4  
```

Comment:
This should read "The list of certificate issuer thumbprints includes but 1fb86b1168ec743154062e8c9cc5b171a4b7ccb4 not 626d44e704d1ceabe3bf0d53397464ac8080142c." 

There may be other issuer TPs alongside 1fb..ccb4.